robocup_ws/src/Planner/src/player_controller.py

```diff
@@ -282,5 +421,62 @@ def get_action(self, my_pos_efcs:Position):
             self.current_goal += 1
             self.current_goal %= 6
         l_rpm, r_rpm, = simple_go_to_action(my_pos_efcs, goal_pos)
-        print(f"({l_rpm}, {r_rpm})")
-        return PlayerCommand(l_rpm, r_rpm, 0)
\ No newline at end of file
+        return PlayerCommand(l_rpm, r_rpm, 0)
+
+    def distance_judge(self, game_info: list, init_distance = None) -> [bool, int, int]:
```

Comment:
This can be shared between teams

Reply:
for now team will stay as is because it is dumb..no?

Reply:
yeah... not fully convinced, but skip this